### ISSUE TITLE
Mention 4.16 TCP Tracepoints

### ISSUE THREAD

**Issue by zoidbergwill** [zoidbergwill/awesome-ebpf#7]
http://www.brendangregg.com/blog/2018-03-22/tcp-tracepoints.html

**Comment by qmonnet** [comment]
I don't believe it should be the objective of the list to mention or list BPF features itself. Maybe wait for some doc, article, presentation or any relevant resource to appear, and then list it here?

**Comment by zoidbergwill** [comment]
That sounds like a great idea. Good point. Maybe the brendangregg blog post is a worthy link then.
